src/Controller/AdminController.php

```diff
@@ -58,7 +58,7 @@ public function index(
                     break;
             }
         } else {
-            $demands = $demandRepository ->findBy(array('status' => array(0, 1)), array('deadline' => 'ASC'));
```

Comment:
Can you change the array() by [] to be in sync with new PSR norms (array() are not used anymore)

Reply:
Fixed